packages/expo-dev-menu/android/src/debug/java/expo/modules/devmenu/DevMenuManager.kt

```diff
@@ -253,6 +256,33 @@ object DevMenuManager : DevMenuManagerInterface, LifecycleEventListener {
     return Bundle.EMPTY
   }
 
+  fun loadFonts(applicationContext: ReactApplicationContext) {
```

Comment:
Can you merge that with https://github.com/expo/expo/blob/main/packages/expo-dev-menu/android/src/main/java/expo/modules/devmenu/modules/internals/DevMenuInternalFontManagerModule.kt?